addons/account/models/account_tax.py

```diff
@@ -1321,7 +1321,7 @@ def _add_tax_details_in_base_line(self, base_line, company, rounding_method=None
         taxes_computation = base_line['tax_ids']._get_tax_details(
             price_unit=price_unit_after_discount,
             quantity=base_line['quantity'],
-            precision_rounding=base_line['currency_id'].rounding,
+            precision_rounding=base_line['currency_id'].rounding or company.currency_id.rounding,
```

Comment:
I think this should be one inside `_get_tax_details` that way we are sure to cover a maximum of cases.